### DISCUSSION TITLE
mpv是不能单独屏蔽某个lua文件吗

### DISCUSSION THREAD

**Discussion by wli189** [dyphire/mpv-config#62] [Q&A] [answered]
我想把mpv作为emby播放器时，单独屏蔽notify_media.lua。但是mpv好像并不能单独实现屏蔽某一个lua。因为这个lua有修改ipc server pipe地址的行为，导致不能回传记录和这个一起使用。

**Comment by dyphire** [comment, marked as answer]
mpv 本身不支持此功能，不嫌麻烦的话可以禁用自动加载脚本功能并手动指定需载入的脚本，参见：
https://github.com/dyphire/mpv-config/blob/5d4f8af91538dde29a899f6481c00342cb838525/mpv.conf#L464-L466
当然最简单的还是直接移除该脚本
